AdyenComponents/BACS Direct Debit/Scenes/Confirmation/BACSConfirmationViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import UIKit
+
+internal protocol BACSConfirmationViewProtocol: FormViewProtocol {
+    func setUserInteraction(enabled: Bool)
+}
+
+internal class BACSConfirmationViewController: FormViewController, BACSConfirmationViewProtocol {
```

Comment:
this class is not needed, since the `presenter` can set itself as the `ViewControllerDelegate`, this `FormViewController` subclass adds no value.